.github/workflows/tests.yml

```diff
@@ -0,0 +1,26 @@
+name: Unit Tests
+
+on: pull_request
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+          cache: 'npm'
+
+      - name: create env file
+        run: |
+          touch .env
+          echo JIRA_USERNAME=${{ secrets.JIRA_USERNAME }} >> .env
+          echo JIRA_TOKEN=${{ secrets.JIRA_TOKEN }} >> .env
+          echo JIRA_HOST='coltdorsey.atlassian.net' >> .env
```

Comment:
I'm pretty sure you can just set the environment variables and not have to add them to a file and that `require('dotenv').config()` will pull them into your modules for use.  See https://docs.github.com/en/actions/learn-github-actions/environment-variables

Reply:
```
 env:
      JIRA_USERNAME: ${{ secrets.JIRA_USERNAME }}
```